p2pserver/block_sync.go

```diff
@@ -342,7 +369,13 @@ func (this *BlockSyncMgr) OnHeaderReceive(headers []*types.Header) {
 	if !this.isHeaderOnFlight(height) {
 		return
 	}
-	err := this.ledger.AddHeaders(headers)
+
+	count := len(headers)
```

Comment:
question:
headers[] has been checked be sequential-continuous?

Reply:
yes